diff --git a/node_modules/@walletconnect/types/index.d.ts b/node_modules/@walletconnect/types/index.d.ts
index 80a6c06..e80b7b8 100644
--- a/node_modules/@walletconnect/types/index.d.ts
+++ b/node_modules/@walletconnect/types/index.d.ts
@@ -335,6 +335,7 @@ export interface IInternalRequestOptions extends IRequestOptions {
 
 export interface ICreateSessionOptions {
   chainId?: number;
+  network?: string;
 }
 export abstract class IEvents {
   public abstract on(event: string, listener: any): void;
